scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -624,7 +624,7 @@ def load_from_file(self, file, filter_platform=[]):
                     instance.metrics['available_ram'] = ts.get('available_ram', 0)
                     instance.metrics['available_rom'] = ts.get('available_rom', 0)
 
-                    status = ts.get('status')
+                    status = ts.get('status', TwisterStatus.NONE)
                     status = TwisterStatus(status) if status else TwisterStatus.NONE
```

Comment:
this line can be removed now

Reply:
I've used part of the https://github.com/zephyrproject-rtos/zephyr/pull/77168 (`None` is recognised as `TwisterStatus.NONE` inside the `TwisterStatus(val)` cast) in order to make the dictionary retrieval and casting more proper.